corehq/apps/domain/templates/domain/manage_releases_by_app_profile.html

```diff
@@ -12,7 +12,6 @@
   {% initial_page_data 'appVersionOnlyShowReleased' True %}
   {% initial_page_data 'buildProfilesPerApp' build_profiles_per_app %}
   {% registerurl 'toggle_release_restriction_by_app_profile' domain '---'%}
-  {% registerurl 'manage_hosted_ccz' domain %}
```

Comment:
Looks like there is some more cleanup to do: https://github.com/dimagi/commcare-hq/search?q=manage_hosted_ccz

Reply:
Oops! [Here's](https://github.com/dimagi/commcare-hq/pull/32541) the cleanup for that.